pages/common/jpegoptim.md

```diff
@@ -13,3 +13,11 @@
 - Force the output images to be progressive:
 
 `jpegoptim --all-progressive {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Force the output images to be fixed max size:
+
+`jpegoptim --size={{250k}} {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Recursive folder optimization:
```

Comment:
Sorry, but we do not usually mention examples like these. It is out of scope for tldr. This falls into the category of cheat sheets.

Rarely we might consider one or two exceptions, but I think this is fairly straightforward to construct, given that `tldr find` already shows the -exec usage.

Reply:
I'm inclined to agree here.

If you're curious for an alternative way of doing that that maxes out all CPU cores though, I've got this one-liner stashed away:

```
find . -regextype egrep -iregex '^.*\.jpe?g$' -print0 | xargs -0 -n1 -P4 jpegoptim --preserve --all-progressive
```

Reply:
@sbrl thank you! 

i changed the page only with --size line